unit-tests/src/test/java/org/eclipse/collections/impl/bag/sorted/immutable/ImmutableSortedBagImplNoIteratorTest.java

```diff
@@ -39,6 +39,30 @@
         return new ImmutableSortedBagImplNoIterator<>(SortedBags.immutable.with(comparator, 1, 1, 1, 2));
     }
 
+    @Test
+    @Override
+    public void streamToBag()
+    {
+    }
```

Comment:
Maybe better to add some comments to explain why we skip stream tests (i.e. using `IteratorSpliterator` in prod code)?